### ISSUE TITLE
Page Transitions Example

### ISSUE THREAD

**Issue by evanwinter** [sveltejs/examples#9]
**Please describe the example**
SvelteKit makes it really easy to do page transitions with `svelte/transition`. It would be cool to have a go-to example for this. [This is how I've been doing it]('https://github.com/evanwinter/sveltekit-page-transitions') but would love to know if there's a better way.

- [x] the example is focused on a specific use case or technology
- [x] the example is self-contained and easy to grasp
- [x] the example is stable and does not require much maintenance

**Comment by karimfromjordan** [comment]
Yeah, this is pretty much how it's done in Svelte with most routers including Sapper and SvelteKit. Routers like Tinro also mention this approach in their READMEs. Something else I've been thinking about for the future is an example for page transitions using the new View Transition API. A few people have already been playing around with it. But that might be better as a separate example until all browsers support the API.
